src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonOutputFormatter.cs

```diff
@@ -25,16 +24,11 @@ public class JsonOutputFormatter : TextOutputFormatter
         // the serializer and invalidate it when the settings change.
         private JsonSerializer _serializer;
 
-        public JsonOutputFormatter()
-            : this(SerializerSettingsProvider.CreateSerializerSettings(), ArrayPool<char>.Shared)
-        {
-        }
-
-        public JsonOutputFormatter(JsonSerializerSettings serializerSettings)
-            : this(serializerSettings, ArrayPool<char>.Shared)
-        {
-        }
```

Comment:
:see_no_evil: Decided creating a customized JSON output formatter is an uncommon scenario and doesn't need to be incredibly simple. The required parameters are easy to create (because `SerializerSettingsProvider` is public) or to grab from DI. The same is true for the JSON input formatters though they have a couple more values that users must grab from DI.